.github/workflows/docker.yml

```
@@ -0,0 +1,62 @@
+
+      - name: Build and Publish Docker Image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          push: ${{ github.event_name != 'pull_request' }}
+          target: release
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          platforms: linux/amd64
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)